packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -891,7 +897,7 @@ export default function parse(source: string, state: State): TemplateParseResult
         el: IRElement,
         pattern: string | RegExp
     ): IRAttribute | undefined {
-        const node = el.__original as parse5.AST.Default.Element;
+        const node = irElementMap.get(el) as parse5.AST.Default.Element;
```

Comment:
lots of occurrences of irElementMap.get(el); Should we preserve the reference to the original element for reuse?

Reply:
I'm not really worried about that since they're all compiler warnings.